recipe/run_conda_forge_build_setup_linux

```diff
@@ -20,8 +20,12 @@ else
     export CPU_COUNT=2
 fi
 
-# Need strict priority for pypy as defaults is not fixed
-if [[ "$CONFIG" == *_pypy* ]]; then
+# Need strict priority for
+# - pypy as defaults is not fixed
+# - cos7 as defaults is not fixed
+# but ppl can turn this off
+cfg_channel_priority=$(grep "channel_priority:" ${FEEDSTOCK_ROOT}/conda-forge.yml )
+if [[ ${cfg_channel_priority} == *channel_priority: strict* ]] || [[ -z ${cfg_channel_priority} ]]; then
```

Comment:
Can you use `shyaml` as in https://github.com/conda-forge/conda-forge-ci-setup-feedstock/blob/master/recipe/run_conda_forge_build_setup_osx#L10

Reply:
that's clever! will change!

Reply:
done